CHANGES.txt

```diff
@@ -1,3 +1,5 @@
+2.5.2 (September 25, 2019)
+ - Update docker base image
```

Comment:
Please add the reason for the update. For example:

- Increase version to patch security vulnerabilities